functions/completed-checks/sample.data.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
@GuyHarwood the following data are not accessible via local storage: `schoolPin `, `pupilPin `, `firstName `, `lastName `, `dob `
Also it seems `audit`, `answers`, `inputs` are missing

Reply:
good catch.  pupil details removed, will add missing data.